### PR TITLE
teste

### PR DESCRIPTION
**Pull request by Ribeirorodrigo05**
CU-866b05cny  CU-866b05cnq

### CODE REVIEW

**Comment by Ribeirorodrigo05** [comment]
Task linked: [CU-866b05cny teste_3](https://app.clickup.com/t/866b05cny)

**Comment by Ribeirorodrigo05** [comment]
Task linked: [CU-866b0583r teste github](https://app.clickup.com/t/866b0583r)

**Comment by Ribeirorodrigo05** [comment]
Task linked: [CU-866b05cnq teste_2](https://app.clickup.com/t/866b05cnq)

**Comment by Ribeirorodrigo05** [comment]
Task linked: [CU-866b05cny teste_3](https://app.clickup.com/t/866b05cny)

**Comment by Ribeirorodrigo05** [comment]
Task linked: [CU-866b05cnq teste_2](https://app.clickup.com/t/866b05cnq)

**Comment by Ribeirorodrigo05** [comment]
Task linked: [CU-866b0583r teste github](https://app.clickup.com/t/866b0583r)

File: src/resolvers/user/mutation/user.js [hunk 1]
```diff
@@ -1,7 +1,5 @@
 export default {
   Mutation: {
-    createUsers: async (obj, args, context, requestInfo) => {
-      console.log(obj, args, context, requestInfo);
-    },
+    createUsers: async (obj, args, context, requestInfo) => {},
```
Comment (JosueFS) [inline src/resolvers/user/mutation/user.js:3]: Muito clean, falta um console.log


```suggestion
    createUsers: async (obj, args, context, requestInfo) => {
        console.log("Debugando");
    },
```

**Pull request merged**